tools/smoke-tests/error-prone-checker-tests/src/test/java/io/spine/test/tools/check/VariousValidMessageUsages.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.test.tools.check;
+
+import com.google.protobuf.Empty;
+import com.google.protobuf.Message;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DisplayName("UseValidatingBuilder check should")
+class VariousValidMessageUsages {
+
+    @Test
+    @DisplayName("pass")
+    void pass() {
+        assertTrue(true, "This class must compile in order for the test to pass.");
+        Empty empty = Empty.getDefaultInstance();
+        assertEquals(generic(empty), empty);
+    }
+
+    /**
+     * Usage or {@code toBuilder()} methods with generic types is allowed, since there is no
+     * abstract base with the {@code toVBuilder()} method.
+     */
+    @SuppressWarnings("unused")
```

Comment:
I also do not understand the suppression. The `value` is used. The method is called. The `toBuilder()` return result is used.

Also, you seem to comment on a different issue. Or is this a method documentation?